site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,100 @@
+some cases these behave differently in extensions.
+
+This article does not cover extension specific APIs. For more on these, see
+[chrome.storage][chrome-storage-api].
+
+## Storage {: #storage }
+
+### Persistence {: #storage-persistence }
+
+Extension storage is always persistent. For example, when a user uses the "Clear Browsing Data"
+feature, this does not affect extensions. This includes any data stored using web storage APIs (such
+as Local Storage and IndexedDB). Additionally, extensions are not subject to storage eviction under
+heavy memory pressure.
+
+By default, extensions are subject to the normal quota restrictions on storage, which can be checked
+using the `navigator.storage.estimate()` method. However, they can request the `unlimitedStorage`
+permission which affects both extension and web storage APIs.
+
+### Access in service workers {: #storage-in-service-workers }
+
+Some web storage APIs like IndexedDB are accessible in service workers. However, Local Storage and
+Session Storage are not.
+
+Instead, use an [offscreen document][offscreen]. For example, you could use the following to migrate
+data when updating from Manifest V2 to Manifest V3:
+
+1. In the extension service worker check `chrome.storage` for your data.
+1. If your data isn't found, [create][create-offscreen] an offscreen document and call
+[`sendMessage()`][send-message] to start a conversion routine. Read the data from the web storage
+APIs and use message passing to return it to the service worker.
+1. In the service worker, write the data to a new location.
+
+### Partitioning {: #storage-partitioning }
+
+To prevent certain types of cross-site tracking, [storage partitioning][storage-partitioning]
+creates a seperate storage area in certain third-party contexts. In practice, this means that
+if site A embeds an iframe containing site B, site B will not be able to access the same storage
+it would usually have when navigated to directly.
+
+To mitigate the impact of this in extensions, two key exemptions apply:
+
+- If a page with the chrome-extension:// scheme is embedded in any site, storage partitioning will
+not apply, and the extension will have access to its normal storage partition.
+- If a page with the chrome-extension:// scheme includes an iframe, and the extension has
+[host permissions][declare-permissions] for the site it is embedding, that site will also have
+access to its normal storage partition.
+
+Combined, these mean that storage partitioning is not usually something which extension developers
+should need to consider.
+
+## Cookies {: #cookies }
+
+### Secure cookies {: #secure-cookies }
+
+The [`Secure`][cookies-restrict-access] cookie attribute is only supported for the https:// scheme.
+Consequently, chrome-extension:// pages are not able to set cookies with this attribute. This also
+means that extensions cannot use the [`Partitioned`][chips] attribute, which requires that the
+`Secure` attribute is also set.
```

Comment:
Are there any implications of this for extensions?

Reply:
I don't think so. It means they can't create partitioned cookies, but that's a new feature, so it's not something extension developers are used to having. And I would argue cookies aren't really a good storage mechanism to use in extensions.